tests/workflow_tests/test_nph_sms_workflows.py

```diff
@@ -457,3 +457,25 @@ def test_sms_pipeline_n1_function(self, task_mock):
                                      payload=data,
                                      queue='nph')
 
+
+    def test_sms_ingestion_filetype_failure(self):
+
+        self.create_cloud_csv("test_sample_list_fail.txt", "test_sample_list_fail.txt")
+        ingestion_data = {
+            "job": "FILE_INGESTION",
+            "file_type": "SAMPLE_LIST",
+            "file_path": f"{self.test_bucket}/test_sample_list_fail.txt"
+        }
+        workflow = SmsWorkflow(ingestion_data)
+        workflow.execute_workflow()
+
+        from rdr_service.resource import main as resource_main
+        with self.assertLogs(level="WARNING") as cm:
```

Comment:
sry, what is cm? I also am seeing it in some other functions but I am not sure what it is.

Reply:
My understanding of the with statement there is to store the logging output so it can be asserted against for level and content. cm is the variable used elsewhere, but I can update that to something more suitable to this. I am using it to check that I got a warning(not error) log and that it contains the proper warning that shows it reached the point in the code where a slack message would be sent and the ingestion process would stop.